### PR TITLE
Mqtt light command templates

### PR DESCRIPTION
**Pull request by orosam**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: home-assistant/core/pull/84988
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/mqtt.markdown [hunk 1]
```diff
@@ -308,6 +308,7 @@ Configuration variable names in the discovery payload may be abbreviated to cons
     'hold_stat_tpl':       'hold_state_template',
     'hold_stat_t':         'hold_state_topic',
     'hs_cmd_t':            'hs_command_topic',
+    'hs_cmd_tpl':          'hs_command_template',
```
Comment (jbouwh) [inline source/_integrations/mqtt.markdown:311]: Did you miss the abbreviation for `xy_command_template`?

**Review by jbouwh — CHANGES_REQUESTED** [review]

File: source/_integrations/light.mqtt.markdown [hunk 2]
```diff
@@ -217,14 +217,18 @@ effect_value_template:
...
+  type: string
 hs_command_topic:
   description: "The MQTT topic to publish commands to change the light's color state in HS format (Hue Saturation).
   Range for Hue: 0° .. 360°, Range of Saturation: 0..100.
   Note: Brightness is sent separately in the `brightness_command_topic`."
   required: false
   type: string
 hs_state_topic:
-  description: "The MQTT topic subscribed to receive color state updates in HS format.
+  description: "The MQTT topic subscribed to receive color state updates in HS format. The expected payload is the hue and saturation values separated by commas, for example, `359.5,100.0`.
```
Comment (jbouwh) [inline source/_integrations/light.mqtt.markdown:231]: May be you can do the same addition for `xy_state_topic`

File: source/_integrations/mqtt.markdown (hunk 1 above)
Comment (orosam) [reply source/_integrations/mqtt.markdown:311]: Yes, I missed it, thanks. Done.

File: source/_integrations/light.mqtt.markdown (hunk 2 above)
Comment (orosam) [reply source/_integrations/light.mqtt.markdown:231]: And I missed that, too. The XY-related code was added later. Added, thanks.

**Review by jbouwh — APPROVED** [review]
Looks good to me,
Thnx @orosam !

**Pull request merged**